enterprise/backend/test/metabase_enterprise/sandbox/api/user_test.clj

```diff
@@ -76,7 +76,7 @@
 
     (testing "404 if user does not exist"
       (is (= "Not found."
-             (mt/user-http-request :crowberto :put 404 (format "mt/user/%d/attributes" (inc (t2/count 'User))) {}))))
```

Comment:
Fix the flaky `update-user-attributes-test` 

https://github.com/metabase/metabase/actions/runs/4320997119/jobs/7541780631#step:5:1042